package.json

```diff
@@ -1,56 +1,59 @@
-  "main": "distribution/index.js",
-  "scripts": {
-    "build": "babel source --out-dir distribution",
-    "lint": "./node_modules/.bin/eslint source/ test/",
-    "test": "npm run lint && npm run karma",
-    "karma": "karma start --single-run",
-    "karma:watch": "karma start"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/d4l-data4life/js-crypto.git"
-  },
-  "keywords": [
-    "crypto"
-  ],
-  "author": "D4L data4life gGmbH",
-  "license": "ISC",
-  "bugs": {
-    "url": "https://github.com/d4l-data4life/js-crypto/issues"
-  },
-  "homepage": "https://www.d4l.io/",
-  "dependencies": {
-    "base64-js": "^1.3.0",
-    "fast-text-encoding": "^1.0.0",
-    "pbkdf2": "^3.0.16"
-  },
-  "devDependencies": {
-    "@babel/cli": "^7.5.5",
-    "@babel/core": "^7.5.5",
-    "@babel/polyfill": "^7.0.0-beta.54",
-    "@babel/preset-env": "^7.5.5",
-    "babel-eslint": "^8.2.6",
-    "babelify": "^9.0.0",
-    "browserify": "^16.3.0",
-    "chai": "^4.2.0",
-    "eslint": "^5.16.0",
-    "eslint-config-airbnb": "^14.1.0",
-    "eslint-config-airbnb-base": "^11.3.2",
-    "eslint-import-resolver-configurable": "^0.1.3",
-    "eslint-plugin-import": "^2.18.2",
-    "karma": "6.3.2",
-    "karma-babel-preprocessor": "^8.0.1",
-    "karma-browserify": "^7.0.0",
-    "karma-chai": "^0.1.0",
-    "karma-chrome-launcher": "^3.1.0",
-    "karma-coverage": "^2.0.2",
-    "karma-mocha": "^2.0.1",
-    "karma-mocha-reporter": "^2.2.5",
-    "mocha": "^7.2.0",
-    "watchify": "^3.11.1"
-  }
+    "name": "@d4l/js-crypto",
+    "version": "1.2.1",
+    "description": "The Data4Life JavaScript crypto library used for performing crypto operations using the native Web Crypto API.",
+    "main": "distribution/index.js",
+    "scripts": {
+        "build": "babel source --out-dir distribution",
+        "lint": "eslint --ext .ts source/ ",
+        "test": "npm run lint && npm run karma",
+        "karma": "karma start --single-run",
+        "karma:watch": "karma start"
+    },
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/d4l-data4life/js-crypto.git"
+    },
+    "keywords": [
+        "crypto"
+    ],
+    "author": "D4L data4life gGmbH",
+    "license": "ISC",
+    "bugs": {
+        "url": "https://github.com/d4l-data4life/js-crypto/issues"
+    },
+    "homepage": "https://www.d4l.io/",
+    "dependencies": {
+        "base64-js": "^1.3.0",
+        "fast-text-encoding": "^1.0.0",
+        "pbkdf2": "^3.0.16"
+    },
+    "devDependencies": {
+        "@babel/cli": "^7.5.5",
+        "@babel/core": "^7.5.5",
+        "@babel/polyfill": "^7.0.0-beta.54",
+        "@babel/preset-env": "^7.5.5",
+        "@types/base64-js": "^1.3.0",
+        "@types/mocha": "^8.2.3",
+        "@types/pbkdf2": "^3.0.0",
+        "@typescript-eslint/eslint-plugin": "^4.28.3",
+        "@typescript-eslint/parser": "^4.28.3",
+        "babel-eslint": "^8.2.6",
+        "babelify": "^9.0.0",
```

Comment:
Do we still need `babelify` considering the changes at https://github.com/d4l-data4life/js-crypto/pull/18/files#diff-25bdb46a5428d318648ec9eb46a99c0268d8c96b672c46e2247a685bc3e384fbL20?